input/script_utils.go

```diff
@@ -573,31 +581,107 @@ func SenderHTLCTapLeafSuccess(receiverHtlcKey *btcec.PublicKey,
 	return txscript.NewBaseTapLeaf(successLeafScript), nil
 }
 
+// htlcType is an enum value that denotes what type of HTLC script this is.
+type htlcType uint8
+
+const (
+	htlcLocalIncoming htlcType = iota
```

Comment:
Missing docs, which I think is important since these types can easily become confusing.